src/envs/mountain_car.jl

```diff
@@ -6,9 +6,17 @@ module MountainCarEnv
 
 using Reinforce: AbstractEnvironment
 using LearnBase: DiscreteSet
-using Random: srand
 using RecipesBase
 using Distributions
+@static if v"0.7" <= VERSION < v"1.0"
+  using Random: srand
+  seed! = srand
+elseif VERSION >= v"1.0"
+  using Random: seed!
+else
+  seed! = srand
+end
+
```

Comment:
Both 0.7 and 1.0 are `Random.seed!`.
Invoking `Random.srand` will pop the deprecation warning up.

Reply:
oh, you're right, i don't know how I got the impression that it wasn't.